examples/security/graphdb-users.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: graphdb-users
+data:
+  users.js: {{ .Files.Get "files/users.js" | b64enc | quote }}
```

Comment:
How do you expect to resolve this `files/` ? 
To simplify the example, use `stringData` instead of `data` and put the JSON here.